src/seedu/addressbook/ui/TextUi.java

```diff
@@ -83,39 +84,27 @@ public String getUserCommand() {
             fullInputLine = in.nextLine();
         }
 
-        showToUser("[Command entered:" + fullInputLine + "]");
+        out.print(new Formatter().showUserCommand(fullInputLine));
```

Comment:
instead of keep having `new Formatter()`, you can create and use the `formatUi` property?